gatein/src/main/java/org/exoplatform/platform/qa/ui/gatein/pageobject/MyDashBoard.java

```diff
@@ -26,21 +36,14 @@ public MyDashBoard(TestBase testBase) {
   /**
    * Add a gadget to dashboard
    *
-   * @param name the name of gadget that will be added to dashboard
-   * @param numberCol the number of column that will put the gadget. We have 3
-   *          columns as: + numberCol= 1: this is for left column + numberCol= 2:
-   *          this is for middle column + numberCol =3: this is for right column
    */
-  public void addGadget(String name, String numberCol) {
+  public void addGadget() {
     info("Click on GadGet button");
     evt.click(ELEMENT_MYDASH_BTN_ADDGADGET);
     evt.waitForAndGetElement(ELEMENT_DASHBOARD_WORKSPACE_POPUP_TITLE, 2000, 0);
     info("The popup is shown");
-    evt.dragAndDropToObject(ELEMENT_MYDASH_GADGET_NAME.replace("${name}", name),
-                            ELEMENT_MYDASH_GADGET_COLUMN.replace("${number}", numberCol));
-
+    $(byXpath("//*[@class='GadgetTitle' and @title='Services Management']")).dragAndDropTo($(byXpath("//div[@id=\"GadgetContainer\"]"))).waitUntil(Condition.visible,Configuration.timeout);
```

Comment:
The previous version of this method allowed to add any gadget. With this change, we only can add the gadget "Services Management". Is it intended ? Shouldn't we keep the generic version ?